component/class/defaults.yml

```diff
@@ -168,6 +168,16 @@ parameters:
                 # If the alert is pending for more than 5m this indicates a real problem.
                 for: 6m
               ticket_alert: {}
+        minio:
+          uptime:
+            objective: 99.9
```

Comment:
I've used the same values as redis/postgres, any objections?

Reply:
since we haven't got any special requirements, I'd keep it as default